src/main/scala/wdl4s/WdlNamespace.scala

```diff
@@ -419,13 +424,15 @@ object WdlNamespace {
 
   private def declarationName(declarationAst: Ast): Terminal = declarationAst.getAttribute("name").asInstanceOf[Terminal]
 
+  def referentiallyUnsoundValues(container: Scope, expression: WdlExpression): Iterable[Terminal] =
```

Comment:
A little bit of commenting would be helpful here, I wasn't initially sure what "referentially unsound" meant in this context.